File-Formats/DocIO/html.md

```diff
@@ -278,6 +278,103 @@ N> Calling the above event is mandatory in ASP.NET Core, UWP, and Xamarin platfo
 
 * [How to get image from URL while opening HTML in .NET Core targeting applications?](https://www.syncfusion.com/kb/13053/how-to-get-image-from-url-while-opening-html-in-asp-net-core)
 
+### Customize image Path
+
+DocIO provides an [ImageNodeVisited](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_ImageNodeVisited) event, which is used to customize the image path that is set in the output HTML file and save images externally while converting a Word document to HTML.
+
+The following code example illustrates how to save image files during a Word to HTML conversion.
+
+{% tabs %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Open the file as a Stream.
+using (FileStream docStream = new FileStream("Data/Input.docx", FileMode.Open, FileAccess.Read))
+{
+    //Load the file stream into a Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Hook the event to customize the image. 
+        document.SaveOptions.ImageNodeVisited += SaveImage;
+        using (FileStream outputStream = new FileStream("WordtoHTML.html", FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite))
+        {
+            //Save the HTML file.
+            document.Save(outputStream, FormatType.Html);
+        }
+    }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Open an existing Word document. 
+using (WordDocument document = new WordDocument("Input.docx"))
+{
+    //Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage;
+    //Save a Word document as a HTML file.
+    document.Save("WordtoHTML.html", FormatType.Html);
+}
+
+{% endhighlight %}
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Open an existing Word document. 
+Using document As WordDocument = New WordDocument("Input.docx")
+    'Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage
+    'Save a Word document as a HTML file.
+    document.Save("WordtoHTML.html", FormatType.Html)
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+The following code example illustrates the event handler for customizing the image path and saving the image in an external folder
```

Comment:
The following code example illustrates the event handler for customizing the image path and saving the image in an external folder.

Reply:
modified